pkg/operator/ceph/object/user/controller.go

```diff
@@ -433,7 +541,7 @@ func generateUserCaps(user admin.User) string {
 	return caps
 }
 
-func generateUserConfig(user *cephv1.CephObjectStoreUser) *admin.User {
+func (r *ReconcileObjectStoreUser) generateUserConfig(user *cephv1.CephObjectStoreUser) (*admin.User, error) {
```

Comment:
Am I reading correctly here? It looks like `error` return was added, but now the only usage is to return `nil` at the end of this function? Seems like the error return can be removed again. It also seems like the `r *ReconcileObjectStoreUser` receiver isn't in use either? Are these still important changes to keep?

Reply:
That is left over from before usage of the k8s api was removed from the method. I'll revert it back to a func.